app/app/api/private-cloud/products/[licencePlate]/comments/_operations/create.ts

```diff
@@ -12,9 +14,16 @@ export async function createOp(text: string, userId: string, projectId?: string,
     requestId: requestId || undefined,
   };
 
-  const comment = await prisma.privateCloudComment.create({
-    data,
-  });
+  try {
```

Comment:
Let's avoid using `try` here so the error can be caught at a higher level as `5xx` error.